certif/tests/unit/models/certification-issue-report_test.js

```diff
@@ -61,7 +61,7 @@ module('Unit | Model | certification issue report', function (hooks) {
   });
 
   test('it should return the right code for the subcategory', function (assert) {
-    assert.expect(11);
+    assert.expect(12);
```

Comment:
**Suggestion** : Peut être pas pertinent de laisser ça ici, ça n'aide pas la lecture du test.

Reply:
Ca permet de compte le nombre d'assert attendue (utile pour la boucle). Par contre je l'expliciterais bien:
```suggestion
    const expectedAssertNumber = 12;
    assert.expect(expectedAssertNumber);
```

Reply:
J'ai explicité.